### PR TITLE
update white-space demo2es6

### PR DESCRIPTION
**Pull request by iyutao**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

#824

### CODE REVIEW

File: components/white-space/demo/basic.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ const PlaceHolder = props => (
   >Block</div>
 );
 
-const WhiteSpaceExample = React.createClass({
+class WhiteSpaceExample extends React.Component {
```
Comment (paranoidjk) [inline components/white-space/demo/basic.md]: 这种stateless的组件写成pure function更佳一点 https://github.com/ant-design/ant-design-mobile/pull/849

Comment (afc163) [inline components/white-space/demo/basic.md]: https://github.com/ant-design/ant-design-mobile/blob/35344712f8046e6e46be8f8ec8aa4aebcfc10931/.eslintrc.js#L30-L31

Comment (paranoidjk) [inline components/white-space/demo/basic.md]: OK，i will do it later

Comment (iyutao) [inline components/white-space/demo/basic.md]: 嗯，发现了，已经着手改了

**Pull request merged**